src/ape/__init__.py

```diff
@@ -4,93 +4,23 @@
 
```

Comment:
Because of the changes in this file, the docs no longer make a lot of sense.

We _have_ to document the top-level imports here...

So you may need to do like this:

```python
from .managers import acconts as _accounts

accounts = _accounts
"""
<TOP-LEVEL ACCOUNTS DOCS>
"""
```